### PR TITLE
Sample code for the article on set comprehension

### PR DESCRIPTION
**Pull request by lpozo**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: python-set-comprehension/colors.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+colors = {"blue", "red", "green", "orange"}
```
Comment (gahjelle) [inline python-set-comprehension/colors.py:1]: Should we add the duplicate `green` for consistency?

File: python-set-comprehension/emails.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+emails = [
```
Comment (gahjelle) [inline python-set-comprehension/emails.py:1]: Should we start with a set here, like we now do in the tutorial?

File: python-set-comprehension/tools.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+tools = ["Python", "Django", "Flask", "pandas", "NumPy"]
+tools_set = {tool.lower() for tool in tools}
+
+print(tools_set)
+print("python".lower() in tools_set)
+print("Pandas".lower() in tools_set)
+print("Numpy".lower() in tools_set)
+print("Numpy".lower() in tools_set)
```
Comment (gahjelle) [inline python-set-comprehension/tools.py:8]: There is a duplicate line here

**Review by gahjelle — COMMENTED** [review]
Hi @lpozo 

I have a few minor comments which I think are all due to changes we did during TR :)

**Review by brendaweles — APPROVED** [review]

**Pull request merged**